keyboards/install.php

```diff
@@ -252,14 +230,56 @@ protected static function WriteLinuxBoxes() {
     }
 
     protected static function WriteAndroidBoxes() {
-      return self::download_box(
-        'Keyman for Android',
-        PlayStore::url,
-        htmlentities(self::$keyboard->name) . ' for Android',
-        'Installs only ' . htmlentities(self::$keyboard->name) . '. <a href="'.PlayStore::url.'">Keyman for Android</a> must be installed first.',
-        'download-android',
-        'Install on Android',
-        'android');
+      $keyboard = self::$keyboard;
+      $tier = self::$tier;
+      $version = self::$versions->android->$tier;
+
+      $e = [
+        'name' => $keyboard->name,
+        'host' => KeymanHosts::Instance()->downloads_keyman_com,
+        'tier' => $tier,
+        'version' => $version,
```

Comment:
```suggestion
        'version' => $keyboard->version,
```

Since `$version`  is the Keyman for Android version for the tier, I was getting a non-existent keyboard link
https://downloads.keyman.com/keyboards/sil_ipa/13.0.6218/sil_ipa.kmp

Do we still need `$version = self::$versions->android->$tier` if the "Install Keyman for Android" link goes straight to the Play Store?

Reply:
Good pick! Fixed.

Reply:
Also fixed on iOS PR #184 and for Linux and macOS pages.